examples/encoding/plot_rerp_encoder.py

```diff
@@ -0,0 +1,65 @@
+"""
+========================================
+Regression on continuous event-related data (rER[P/F])
+========================================
```

Comment:
Just a quick FYI the examples still have changes needed, they're mostly for
"is this heading in the right direction" purposes for now

Sent from my phone, apologies for the curtness and type-os

On Jul 20, 2016 4:40 PM, "Alexandre Gramfort" <notifications@github.com>
wrote:

> In examples/encoding/plot_rerp_encoder.py
> <https://github.com/mne-tools/mne-python/pull/3310#discussion_r71611921>:
>
> > @@ -0,0 +1,65 @@
> > +"""
> > +========================================
> > +Regression on continuous event-related data (rER[P/F])
> > +========================================
>
> this will not render fine
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3310/files/737be2cd87f8dd485ab37567f6c6b867d85b623e#r71611921>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABwSHVbZLYVU8X1CwuQ9nUnfgZpXHmudks5qXpWvgaJpZM4I28Ut>
> .
>